### ISSUE TITLE
itemProp support on <Meta />

### ISSUE THREAD

**Issue by Jarred-Sumner** [tizmagik/react-head#79]
TLDR: `<Meta />` appears to not have support for the `itemprop` prop, which can be a great tool for improving SEO.

Problem: `<Meta />` tags that have an `itemProp` set don't get added to the DOM.

Ask: Can you add support for the `itemprop` attribute on `<Meta />`? Alternatively, would you accept a PR if I did?

More context:

`<meta />` supports the [`itemprop`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemprop#Same_structured_data_marked_up_in_two_different_ways) attribute, which is a way of annotating markup so that Google can know what a specific element corresponds to. Typically, you use it on the element it the text directly pertains to -- so you'd have `itemProp="datePublished"` on the date published timestamp. 

Sometimes, you can't. Not all metadata is displayed in the page. In that case, you can provide a `<meta itemprop="microdata_attribute_name" content="microdata_attribute_value" />` tag which lets you invisibly add the meta tag, so that Google still knows about the metadata even though it's not actually visible to the user in the same format.

**Comment by Jarred-Sumner** [comment]
actually this is wrong

`itemProp` needs to be near the element so...nevermind
